tap_recharge/client.py

```diff
@@ -131,14 +146,20 @@ def check_access_token(self):
         response = self.__session.get(
             # Simple endpoint that returns 1 record w/ default organization URN
             url='https://api.rechargeapps.com',
-            headers=headers)
+            headers=headers,
+            timeout=self.request_timeout)
         if response.status_code != 200:
             LOGGER.error('Error status_code = %s', response.status_code)
             raise_for_error(response)
         else:
             return True
 
-
+    # Backoff for 5 times when Timeout error occurs
+    @backoff.on_exception(
+        backoff.expo,
+        Timeout,
+        max_tries=5,
+        factor=2)
     @backoff.on_exception(
```

Comment:
@namrata270998 Can we merge Timeout in the existing backoff 
```suggestion
    @backoff.on_exception(
        backoff.expo,
        (Timeout, Server5xxError, requests.ConnectionError, Server429Error),
```

Reply:
> @namrata270998 Can we merge Timeout in the existing backoff

Merged the Timeout backoff in existing backoffs